ci-operator/step-registry/openshift/e2e/libvirt/test/openshift-e2e-libvirt-test-ref.yaml

```diff
@@ -0,0 +1,55 @@
+ref:
+  as: openshift-e2e-libvirt-test
+  from: tests
+  commands: openshift-e2e-libvirt-test-commands.sh
+  env:
+  - name: TEST_TYPE
+    default: "suite"
+    documentation: |
+      The type of test to perform.
+      * 'suite' - Run the test suite defined by TEST_SUITE. By default this executes the
+                  OpenShift parallel conformance suite. All product components must run the
+                  default suite on pull requests prior to merge.
+      * 'conformance-parallel' - Runs the test suite defined by TEST_SUITE, then runs the product conformance/parallel
+                  suite to verify the cluster is still functional. This is typically used for testing
+                  disruption from one suite, then confirming that the disruption did not degrade the
+                  product.
+      * 'conformance-serial' - Runs the test suite defined by TEST_SUITE, then runs the product conformance/serial
+                  suite to verify the cluster is still functional. This is typically used for testing
+                  disruption from one suite, then confirming that the disruption did not degrade the
+                  product.
+      * 'upgrade' - Perform an upgrade to the image defined by OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE.
+                  The TEST_UPGRADE_OPTIONS flag may be optionally set. All product components
+                  must run the default upgrade test (verify an upgrade completes) on pull requests
+                  prior to merge.
+      * 'jenkins-e2e' - Runs the test suite defined by TEST_SUITE, then runs the product jenkins-e2e
```

Comment:
This is not a valid suite for multi-arch. It need to be the `-rhel-only` equivalent.